torch/nn/modules/conv.py

```diff
@@ -376,7 +376,8 @@ class ConvTranspose1d(_ConvTransposeMixin, _ConvNd):
     composed of several input planes.
 
     This module can be seen as the gradient of Conv1d with respect to its input.
-    It is sometimes (but incorrectly) refered to as a deconvolutional operation.
+    It is also known as a fractionally-strided convolution, an à trous convolution, or
```

Comment:
Yes, got a bit too enthusiastic :P Tidied up the conv docs a bit too while I fixed this.